workerPool/pool.go

```diff
@@ -1,74 +0,0 @@
-package workerPool
-
-import (
-	"fmt"
-	"runtime"
-	"sync"
-	"time"
-)
-
-var WorkerQueue chan WorkRequest
-var startPool sync.Once
-var getPoolData sync.Once
-
-type Pool struct {
-	NumOfWorkers int
-	WorkQueue    chan WorkRequest
-	CPUs         int
-	Duration     time.Duration
-}
-
-func GetPoolConfig(noOfWorkers, numCPU int, duration time.Duration) *Pool {
```

Comment:
this is not actually a pool config, we are actually returning a pool object from here, let's rename this to `GetPool` and also we can perform Initialize action in sync.Do

Reply:
Doing it in sync.Do already.